src/platform/Kconfig

```diff
@@ -499,4 +499,13 @@ config XTENSA_EXCLUSIVE
 	  This has to be selected for xtensa exclusive instructions.
 	  There is a definition for EXCLUSIVE option in xtensa-config.h
 
+config FORCE_DMA_COPY_WHOLE_BLOCK
+	bool
+	default y if MT8195
+	default n
```

Comment:
isn't "no" the default default? Could you try a defconfig for another `HOST_PTABLE` platform?

Reply:
I've tested `byt` and the config is indeed undefined.